Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM centos:7
+
+RUN yum install -y java-1.8.0-openjdk-devel make gcc-c++
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
+
+ARG HADOOP_VERSION
+ARG HADOOP_USER_NAME
+ARG ZOOKEEPER_VERSION
+
+ENV HADOOP_VERSION ${HADOOP_VERSION:-2.7.3}
```

Comment:
Seems like the Apache mirrors don't have 2.7.3 anymore; might want to change this to 2.7.4.

Reply:
Sounds good.  I will update to 2.7.4

Reply:
Fixed in bacfe36bd